frontend/src/components/Table.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useTable } from 'react-table';
```

Comment:
I think we want to put comments on every import

Reply:
Thanks, will update!